proto-compiler/src/cmd/compile.rs

```diff
@@ -104,7 +104,6 @@ impl CompileCmd {
             .compile_well_known_types(true)
             .client_mod_attribute(".", r#"#[cfg(feature = "client")]"#)
             .build_server(false)
-            .format(true)
```

Comment:
Nevermind, that's why we are seeing formatting changes in the generated code.

What's the rationale for disabling formatting here by the way?

Reply:
The flag is no longer supported by prost-build.
I think the formatting is on by default now; before, if you don't enable it, the generated source came out human-unreadable without pretty whitespace.